docs/ref/templates/api.txt

```diff
@@ -146,14 +146,20 @@ what's passed by :class:`~django.template.backends.django.DjangoTemplates`.
 
 .. staticmethod:: Engine.get_default()
 
-    When a Django project configures one and only one
-    :class:`~django.template.backends.django.DjangoTemplates` engine, this
-    method returns the underlying :class:`Engine`. In other circumstances it
-    will raise :exc:`~django.core.exceptions.ImproperlyConfigured`.
+    Returns the the underlying :class:`Engine` from the first configured
+    :class:`~django.template.backends.django.DjangoTemplates` engine. If no
+    engines are configured will raise
+    :exc:`~django.core.exceptions.ImproperlyConfigured`.
 
     It's required for preserving APIs that rely on a globally available,
     implicitly configured engine. Any other use is strongly discouraged.
 
+    .. versionchanged:: 2.0
+
+        Previously would raise
+        :exc:`~django.core.exceptions.ImproperlyConfigured` if multiple engines
+        were configured.
```

Comment:
Analogous to the examples in the ["Documenting new features" documentation](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features), I'd describe the new behavior rather than the old one, e.g.
```rst
    .. versionchanged:: 2.0

        Permitted multiple engines to be configured, so that
        :exc:`~django.core.exceptions.ImproperlyConfigured` isn't raised anymore
        if multiple engines are configured.
```
or even just
```rst
    .. versionchanged:: 2.0

        Permitted multiple engines to be configured.
```
or
```rst
    .. versionchanged:: 2.0

        Permitted multiple engines to be configured, returning the first one if there is more than one.
```